meta/group-info/fr.yaml

```diff
@@ -1,13 +1,64 @@
-outdated: true # missing entries
 separator: ", "
+alumni:
+  roles:
+    academy: fondateur de osu!academy
+    admin: administration
+    chat: Modérateur de tchat
+    dev: osu!dev
+    media: créateur de osu!media
+    support: Équipe de soutien
+    tournaments: Gestionnaire des tournois
 gmt:
+  all_mods: tous les modérateurs
   areas:
-    beatmap: modération du beatmaps
-    chat: modération du chat
-    forum: modération du forum
-    team: équipe officielle
-    dev: développement d'osu!
-    player: support du joueur
+    beatmap: modérateur des beatmaps
+    chat: modérateur du tchat
+    forum: modérateur du forum
+    team: Équipe de présence officiel
+    dev: osu! development
+    player: support au joueur
     support: support technique
-    tournaments: organisation de tournois
+    tournaments: Gestionnaire des tournois
     wiki: administration du wiki
```

Comment:
you should put capital letters on each new line, or not put them everywhere

Reply:
C'est de base dans la version anglaise, je laisse comme tel, j'ai juste remit tout en minuscule